lib/rubocop/name_similarity.rb

```diff
@@ -12,6 +12,8 @@ def find_similar_name(target_name, names)
     end
 
     def find_similar_names(target_name, names)
+      return [] unless defined?(DidYouMean::SpellChecker)
```

Comment:
It would be helpful to link a reference (#8143) to the reason for the workaround :-)

Reply:
Good point. I will add a commit.